### ISSUE TITLE
[IMPROVEMENT: FRONT-END] Re-think website color palette so that it is W3C-WCAG Compliant

### ISSUE THREAD

**Issue by realChakrawarti** [realChakrawarti/ingest#160]
#### The following issue has been raised by @pinpin59 on #159 

- [ ] If the primary color is optimized for general text contrast, buttons using primary-foreground on a primary background no longer meet accessibility standards.

- [ ] Conversely, if we tweak primary to ensure button compliance, general text loses its accessibility compliance.

**It may be necessary to rethink the app’s color palette to find a balance that maintains both accessibility and design consistency. A possible solution could be to separately adjust button colors and global text colors instead of using a single primary variable for both.**

**Comment by duy4nt** [comment]
I would like to help on this issue. I have the google's ux course certificate, and know about accessibility.

**Comment by duy4nt** [comment]
I will have to make some major changes on the color palette as color are failing in the dark mode as well as light mode. 

Can I make a design system file for your website which will contain all the guidelines for the UI, which will be helpful in future?

I will redesign the color palette of your whole website and standardize it in an .md file in a folder called a design_guideline

**Comment by realChakrawarti** [comment]
@duy4nt Please go ahead and feel free to make **major** changes on the color palette. Can't wait to see how it looks post those changes. 😃

 As for design system markdown file or any such design assets, put directly inside `src/app/styles` for now.

**Comment by duy4nt** [comment]
Sounds great. I will get into it asap. thanks for the guidance.

**Comment by duy4nt** [comment]
Hello,
While i was working with the code base, something broke and I am not able to fix it. 
And i dont think that i am the correct person for this task. 
I am providing  you with the design file i made. 

[design_system.md](https://github.com/user-attachments/files/23692383/design_system.md)

Thank you
